source/playbook/index.html.erb

```diff
@@ -17,33 +17,42 @@ directory_index: false
-<p>Parts of the Playbook will be useful to particular stakeholders (e.g., patient organizations, health systems, specialty societies, government agencies, payers, vendors, etc.) and roles (e.g., decision makers, funders, project managers, community organizers, clinical experts, informaticists, standards developers, system engineers, communications personnel, etc.) involved in just a subset of the spectrum of all future work. The <%= link_to("Key Playbook Sections by Role", "/resources/roles.html") %> provided here aims to help guide people with leadership, clinical, or technical responsibilities to quickly find the most relevant information for them.</p>
-<p>
-To get the most out of the Playbook, it would be helpful for leaders to have at least a high-level knowledge of standards that are foundational in the Playbook (e.g., the <a href="https://www.healthit.gov/isa/united-states-core-data-interoperability-uscdi">United States Core Data for Interoperability (USCDI)</a>, <a href="https://hl7.org/fhir/summary.html">Fast Healthcare Interoperability Resources (FHIR)</a>, <a href="https://www.hl7.org/fhir/us/core/">US Core FHIR Implementation Guide</a>, etc.).  Standards implementers should, of course, have a much deeper understanding of these foundations. </p>
+      <p>Parts of the Playbook will be useful to particular stakeholders (e.g., patient organizations, health systems, specialty societies, government agencies, payers, vendors, etc.) and roles (e.g., decision makers, funders, project managers, community organizers, clinical experts, informaticists, standards developers, system engineers, communications personnel, etc.) involved in just a subset of the spectrum of all future work. The <%= link_to("Key Playbook Sections by Role", "/resources/roles.html") %> provided here aims to help guide people with leadership, clinical, or technical responsibilities to quickly find the most relevant information for them.</p>
+      <p>
+      To get the most out of the Playbook, it would be helpful for leaders to have at least a high-level knowledge of standards that are foundational in the Playbook:
+        <ul>
+          <li><a href="https://www.healthit.gov/isa/united-states-core-data-interoperability-uscdi">United States Core Data for Interoperability (USCDI)</a></li>
+          <li><a href="https://hl7.org/fhir/summary.html">Fast Healthcare Interoperability Resources (FHIR)</a></li>
+          <li><a href="https://www.hl7.org/fhir/us/core/">US Core FHIR Implementation Guide</a></li>
+          <li><a href="https://uscdiplus.healthit.gov/uscdi">USCDI+ (specialty supplements to USCDI)<a></li>
+          <li><a href="https://www.aapc.com/resources/what-is-medical-coding">Health Information Coding Systems<a></li>
+          <li><a href="https://www.himss.org/terminology-standards">Health Information Terminology Standards<a>
+        </ul>
+      Standards implementers should, of course, have a much deeper understanding of these foundations. </p>
```

Comment:
While true, I don't think this needs to be said. Recommend just ending with the list of high-level foundational standards.